config/lint/lint.xml

```diff
@@ -4,6 +4,7 @@
     <issue id="MissingTranslation" severity="ignore" />
     <issue id="ExtraTranslation" severity="ignore" />
     <issue id="PluralsCandidate" severity="ignore" /> <!-- GlotPress doesn't support plurals -->
+    <issue id="CredentialDependency" severity="ignore" /> <!-- False positive, fixed in AGP 8.6.0: https://issuetracker.google.com/issues/339841694 -->
```

Comment:
Praise (❤️): 👍 on that and I guess that similarly we can ignore [this](https://github.com/wordpress-mobile/WordPress-Android/security/code-scanning/11376) core scanning security alert.